mediator/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "mediator"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.75"
+aries-vcx = { path = "../aries_vcx" }
+aries_vcx_core = { path = "../aries_vcx_core", features = ["vdrtools_wallet"] }
+async-trait = "0.1.73"
+axum = "0.6"
+axum-macros = "0.3.8"
+cursive = { version = "0.20.0", features = ["crossterm-backend"] }
+diddoc_legacy = { path = "../diddoc_legacy" }
+dotenvy = "0.15"
+env_logger = "0.10.0"
+fast_qr = { version = "0.10.2", features = ["svg"] }
+futures = "0.3.28"
+log = "0.4.20"
+messages = { path = "../messages" }
+reqwest = { version = "0.11.20", features = ["json"] }
+serde = "1.0.188"
+serde_json = "1.0.106"
+sqlx = "0.7"
+# sqlx = { version = "0.5.8", git = "https://github.com/jovfer/sqlx", branch = "feature/json_no_preserve_order_v5" }
+thiserror = "1.0.49"
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+tower-http = { version = "0.4.4", features = ["catch-panic"] }
+url = "2.4.1"
+uuid = "1.4.1"
+xum-test-server = { git = "https://github.com/nain-F49FF806/axum-test-server.git", branch = "aries-vcx-mediator-integration", version = "*" }
```

Comment:
Perhaps I've already wrote this elsewhere before - if yes apologies for duplication - but let's make sure to inline the necessary code from here so we get rid of this dependency.

Reply:
Have inlined the code inside aries-vcx repository.
Currently, it still acts as a local dependency.
As I implement Forward message support, I shall try to integrate it deeper.